crates/gas/src/lib.rs

```diff
@@ -188,7 +193,7 @@ impl From<Gas> for u64 {
 impl Display for Gas {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         // Display the gas in whole amounts
-        write!(f, "{}", self.get_whole_gas_units())
+        write!(f, "{}", self.sub)
```

Comment:
This doesn't convert the gas to whole units anymore so the comment above is misleading. Shouldn't we keep the same logic as before?

Reply:
Ah right this si because we don't have the scale anymore. Probably we should take into consideration removing this trait implementation at this point and create a method with a more descriptive name. Cause I think when we display the gas we really want to display in in whole units, the sub units are of no interest

Reply:
comment changed in [112da61](https://github.com/anoma/namada/pull/3391/commits/112da6116adeb7a1dd41c726aab0232a6b6efd3f)